timemachine/cpp/src/device_buffer.cu

```diff
@@ -32,6 +32,9 @@ template <typename T> void DeviceBuffer<T>::copy_to(T *host_buffer) const {
 }
 
```

Comment:
at some point we should be more precise about these, i.e. use the `uint64_t`, `int64_t`, etc. types from `<cstdint>`